src/main/groovy/life/qbic/portal/sampletracking/ui/SampleList.groovy

```diff
@@ -0,0 +1,62 @@
+package life.qbic.portal.sampletracking.ui
+
+import com.vaadin.data.provider.ListDataProvider
+import com.vaadin.ui.Grid
+import com.vaadin.ui.VerticalLayout
+import groovy.util.logging.Log4j2
+import life.qbic.datamodel.samples.Sample
+
+@Log4j2
+class SampleList extends VerticalLayout {
+    SampleListModel viewModel
+    private Grid<Sample> sampleGrid
+
+    private SampleList() {
+        // prevent default constructor
+        throw new AssertionError()
+    }
+
```

Comment:
```suggestion

    // Suppress default constructor for noninstantiability
    private SampleList() {
        // prevents accidental call from inside the class
        throw new AssertionError()
    }

```

Reply:
I would like to keep the  cryptic `noninstantiability` out of the comments.
```suggestion 
    // Suppress default constructor. Instantiation without parameters not desired.
    private SampleList() {
        // prevents accidental call from inside the class
        throw new AssertionError()
    }

```